crates/history/src/any.rs

```diff
@@ -1,16 +1,12 @@
 use std::borrow::Cow;
 
 #[cfg(feature = "query")]
-use serde::Serialize;
+use crate::{error::HistoryResult, query::ToQuery};
 
-use crate::browser::BrowserHistory;
-#[cfg(feature = "query")]
-use crate::error::HistoryResult;
-use crate::hash::HashHistory;
-use crate::history::History;
-use crate::listener::HistoryListener;
-use crate::location::Location;
-use crate::memory::MemoryHistory;
+use crate::{
+    browser::BrowserHistory, hash::HashHistory, history::History, listener::HistoryListener,
+    location::Location, memory::MemoryHistory,
+};
```

Comment:
Nit: this (and other similar import changes) are needed changes that appears in a diff. Do you have your editor set to forcefully follow this formatting style, perhaps? Rustfmt by default doesn't do this.

If you don't want to revert it, that is fine too. I realized this gloo doesn't have a `rustfmt.toml`. I'll add that after this PR, to avoid introducing conflicts

Reply:
I may have ran some keybinding that runst rustfmt with overriden defaults -- will revert this!